auth-service/src/main/java/com/petclinic/auth/Config/GlobalControllerExceptionHandlerConfig.java

```diff
@@ -2,20 +2,37 @@
 
 import com.petclinic.auth.Exceptions.HTTPErrorMessage;
 import com.petclinic.auth.Exceptions.IncorrectPasswordException;
+import com.petclinic.auth.Exceptions.InvalidInputException;
+import com.petclinic.auth.Exceptions.NotFoundException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import org.springframework.web.context.request.WebRequest;
 
-import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+import static org.springframework.http.HttpStatus.*;
 
 @RestControllerAdvice
 public class GlobalControllerExceptionHandlerConfig {
 
+
     @ExceptionHandler(value = IncorrectPasswordException.class)
     @ResponseStatus(value = UNAUTHORIZED)
     public HTTPErrorMessage resourceNotFoundException(IncorrectPasswordException ex, WebRequest request) {
 
         return new HTTPErrorMessage(401, ex.getMessage());
     }
+
+    @ExceptionHandler(value = NotFoundException.class)
+    @ResponseStatus(value = NOT_FOUND)
+    public HTTPErrorMessage NotFoundException(NotFoundException ex, WebRequest request) {
+
+        return new HTTPErrorMessage(404, ex.getMessage());
```

Comment:
Could use HttpStatus enum for better readability
Check out 
https://github.com/cgerard321/champlain_petclinic/blob/e91d80b9b4689baf6e97dd8374276681178383e8/auth-service/src/main/java/com/petclinic/auth/Config/GlobalControllerExceptionHandlerConfig.java#L28